selection/query_generator.py

```diff
@@ -136,6 +136,36 @@ def _run_command(self, command, return_output=False, shell=False):
     def _files(self):
         return os.listdir(self.directory)
 
+    def _generate_job(self):
+        logging.info("Generating JOB Queries")
+        for filename in os.listdir(self.directory):
+            if ".sql" not in filename or "fkindexes" in filename or "schema" in filename:
+                continue
+            query_id = filename.replace(".sql", "")
+
+            with open(f"{self.directory}/{filename}", "r") as query_file:
+                query_text = query_file.read()
+                query_text = query_text.replace("\t", "")
+                query = Query(query_id, query_text)
+
+                assert "WHERE" in query_text, "Query without WHERE clause encountered"
+
+                split = query_text.split("WHERE")
+                assert len(split) == 2, "Query split for JOB query contains subquery"
+                query_text_before_where = split[0]
+                query_text_after_where = split[1]
+
+                for column in self.columns:
+                    if (
+                        column.name in query_text_after_where
```

Comment:
why not `in query_text`?

Reply:
I believe the reasons was that column names before `WHERE` were not always clearly distinguishable. There are for example multiple tables with columns `info`.

Reply:
Should we add a comment in the code to explain the parsing difficulties and that indexabale columns may be missing.

Reply:
Yes, we should probably do that or implement a more sophisticated parsing solution.

Reply:
I will add a comment to the code and an issue requesting improvements.